src/main/java/com/example/chat/groupchatbackend/ReadStatus.java

```diff
@@ -0,0 +1,65 @@
+package com.example.chat.groupchatbackend;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.time.LocalDateTime;
+
+@Entity
+public class ReadStatus {
+
+    @Id
+    @GeneratedValue
+    private int id;
+    private int conversationId;
```

Comment:
this won't generate foreign key between Conversation and ReadStatus. try something like here: https://stackoverflow.com/a/6311954